src/main/java/core/basesyntax/service/operations/OperationsHandler.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.model.Operations;
+
+public interface OperationsHandler {
+    Operations getOperation();
+
+    int getOperationsResult(int result, int quantity);
```

Comment:
The method name `getOperationsResult` could be simplified to `apply` or `execute` to better reflect its purpose in the context of the Strategy pattern. It's also more intuitive for an operation handler to have a method that implies action.